1001-2000/LIT1990Mawase.xml

```diff
@@ -309,7 +309,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <div type="textpart" xml:id="Haleta">
               <label>Table of melodic variations for the Hallelujah <foreign xml:lang="gez">አንቀጸ፡ ሃሌታ፡</foreign></label>
             </div>
-
+            <div type="textpart" xml:id="LIT6841MawArsima">
```

Comment:
please take a look at the other textparts
`xml:id `is internal ID, custom, can by any string, but it may not start with `LIT`
and then if the textpart has been upgraded to an independent CAe entry because of independent circulation then you also can have a `corresp `which points to that record (in this case `LIT6841MawArsima`)